src/main/java/org/openpnp/machine/reference/vision/wizards/ReferenceBottomVisionConfigurationWizard.java

```diff
@@ -110,6 +110,11 @@ private void editPipeline() throws Exception {
     }
 
     @Override
+    public String getWizardName() {
```

Comment:
If this is shown in the tab it should be "ReferenceBottomVision". If another "BottomVision" type class is added we want to be able to tell them apart. Same with the one below in the wizard.